general/clients/installing-mopidy.md

```diff
@@ -0,0 +1,66 @@
+---
+uid: clients-installing-mopidy
+title: Installing Mopidy Plugin
+---
+
+# Installing Mopidy Plugin
+
+The Mopidy Jellyfin plugin is available to install from [PyPi](https://pypi.org/project/Mopidy-Jellyfin) using pip.
+
+## General
+
+For general use computers, such as workstations or laptops, it's recommended to install Mopidy plugins in user mode.  Installing python packages from pip using sudo or root permissions can lead to conflicts with your package manager in the future.
+
+1. Install Mopidy using your method of choice using the [official docs](https://docs.mopidy.com/en/latest/installation/)
+2. Install the Jellyfin plugin for Mopidy
+    * `pip3 install --user mopidy-jellyfin`
+3. (Optional) Install other mopidy related packages
+    * `pip3 install --user mopidy-mpd mopidy-musicbox-webclient`
+4. Configure your `mopidy.conf` located at `$HOME/.config/mopidy/mopidy.conf`
+  * See [Config File](xref:clients-installing-mopidy#config-file)
+5. There may be a need to install extra gstreamer codecs if they're not already on your system, but these are highly variable and depend on your hardware and distro
+
+## Raspberry Pi (Remote Controlled Speakers)
+
+Utilizing a Raspberry Pi (or other small form factor computer) it's possible to use Mopidy to build a set of standalone smart speakers connected to your Jellyfin server.
+
+1. Grab the latest [raspbian image](https://www.raspberrypi.org/downloads/raspbian/).  Unless you have a need for a GUI, the 'Lite' image is plenty for this project.
+2. Install the image to the sd card (See the [official docs](https://www.raspberrypi.org/documentation/installation/installing-images/README.md))
+3. Install Mopidy from their [apt repo](https://docs.mopidy.com/en/latest/installation/debian/#install-from-apt-mopidy-com) to ensure we get the latest version
+4. Install required OS packages
+    * `sudo apt install mopidy mopidy-mpd gstreamer1.0-plugins-bad python3-pip`
```

Comment:
Code below lists doesn't use bullet points at the moment. I'm still trying to decide the best method, but for now we should keep them consistent with four spaces.